### PR TITLE
Add Python version/implementation classifiers.

### PR DESCRIPTION
**Pull request by jamadden**
This is necessary for tools like @mgedmin 's site https://zope3.pov.lt/py3/ and caniusepython3.com to function correctly.

Also add setup.cfg to indicate that wheels are universal.

### CODE REVIEW

**Comment by mgedmin** [comment]
Tests fail because .travis.yml is bad -- we should be using python: [ 2.7, 3.5, 3.6, pypy ] and tox-travis.

**Comment by jamadden** [comment]
Or avoid tox altogether and take advantage of pip caching.

**Comment by mgedmin** [comment]
tox should be able to take advantage of pip caching, AFAIU -- unless the pip/virtualenv versions in the Travis image are too old?

(The best reason I know not to use tox is when you want to use coverage run + coveralls.)

**Comment by jamadden** [comment]
> tox should be able to take advantage of pip caching, AFAIU -- unless the pip/virtualenv versions in the Travis image are too old?

Last time I checked they generally were.

> (The best reason I know not to use tox is when you want to use coverage run + coveralls.)

Also a good point!

**Comment by jamadden** [comment]
I updated .travis.yml to fix all the tests and to enable coveralls. (Unfortunately coveralls is experiencing some issues right now.)

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+install:
+  - pip install -U pip setuptools
+  - pip install -U coveralls coverage
+  - pip install -U -e ".[test]"
+
+
+cache: pip
+
+before_cache:
+    - rm -f $HOME/.cache/pip/log/debug.log
```
Comment (mgedmin) [inline .travis.yml]: Now everything has 2-space indents except this line ;)

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by mgedmin** [comment]
You have the PyPI bit for zope.app.schema now.

File: .travis.yml (hunk 1 above)
Comment (jamadden) [reply .travis.yml]: Fixed.

**Comment by jamadden** [comment]
Thank you.

**Pull request merged**